src/LiipImagineBundle.php

```diff
@@ -49,7 +49,7 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new ResolversCompilerPass());
         $container->addCompilerPass(new MetadataReaderCompilerPass());
 
-        /** @var $extension LiipImagineExtension */
+        /** @var LiipImagineExtension $extension */
```

Comment:
should we convert the inline `@var` comments to `\assert($extension instanceof LiipImagineExtension);` to trigger warnings when this is not the case?